customtransforms/custom_single_json_retriever.py

```diff
@@ -0,0 +1,52 @@
+from transformer import Transformer
+from utilities import flat
+
+
+class SingleJsonRetriever(Transformer):
+    '''
+    Transforms perfherder data into the standardized data format.
+    '''
+    entry_number = 0
+
+    def transform(self, data):
+        ret = []
+        self.entry_number += 1
+
+        flatted_data = flat(data, [])

```

Comment:
Add a comment here saying something like `# Modify the dict to have one key per dictionary path in the original data`.